packages/web/sentry.server.config.ts

```diff
@@ -11,11 +11,20 @@ Sentry.init({
     process.env.NEXT_PUBLIC_VERCEL_ENV || process.env.NODE_ENV || "development",
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 0.05,
+  tracesSampleRate: 0.01,
 
-  enabled:
-    process.env.NODE_ENV !== "development" && process.env.NODE_ENV !== "test",
+  /**
+   * Disable server-side (Node.js serverless) tracing since we are only interested
+   * in tracing swap transactions, specifically including bridge transfer transactions.
+   */
+  enabled: false,
```

Comment:
For my understanding, what components are encapsulated by this?

Reply:
Currently, we only run the bridge transfer router in Node.js serverless functions because of crypto package dependency. Furthermore, The comment is not clear enough. I'll push a commit clarifying this.